lib/routes/email.js

```diff
@@ -17,14 +22,25 @@ module.exports = {
     }
   },
   handler: function email(req, reply) {
-    var email = req.auth.credentials.email;
-    if (email) {
+    request.get(AUTH_SERVER_URL, {
+      headers: {
+        Authorization: 'Bearer ' + req.auth.token
+      },
+      json: true
```

Comment:
Will this cause the oauth server to have to verity the token twice, once for the profile-server's own scope-checking, then again for the auth-server's permission checks?

If so, I'm OK with it while we're in the process of restructuring things, but we should flag it with @jrgm and @ckolos because they'll see a spike in oauth server traffic.